ie/ie.go

```diff
@@ -11,188 +11,263 @@ import (
+	FramedRoute                                                            uint16 = 153
+	FramedRouting                                                          uint16 = 154
+	FramedIPv6Route                                                        uint16 = 155
+	EventTimeStamp                                                         uint16 = 156
+	AveragingWindow                                                        uint16 = 157
+	PagingPolicyIndicator                                                  uint16 = 158
+	APNDNN                                                                 uint16 = 159
+	TGPPInterfaceType                                                      uint16 = 160
+	PFCPSRReqFlags                                                         uint16 = 161
+	PFCPAUReqFlags                                                         uint16 = 162
+	ActivationTime                                                         uint16 = 163
+	DeactivationTime                                                       uint16 = 164
+	CreateMAR                                                              uint16 = 165
+	AccessForwardingActionInformation1                                     uint16 = 166
+	AccessForwardingActionInformation2                                     uint16 = 167
+	RemoveMAR                                                              uint16 = 168
+	UpdateMAR                                                              uint16 = 169
+	MARID                                                                  uint16 = 170
+	SteeringFunctionality                                                  uint16 = 171
+	SteeringMode                                                           uint16 = 172
+	Weight                                                                 uint16 = 173
+	Priority                                                               uint16 = 174
+	UpdateAccessForwardingActionInformation1                               uint16 = 175
+	UpdateAccessForwardingActionInformation2                               uint16 = 176
+	UEIPAddressPoolIdentity                                                uint16 = 177
+	AlternativeSMFIPAddress                                                uint16 = 178
+	PacketReplicationAndDetectionCarryOnInformation                        uint16 = 179
+	SMFSetID                                                               uint16 = 180
+	QuotaValidityTime                                                      uint16 = 181
+	NumberOfReports                                                        uint16 = 182
+	PFCPSessionRetention                                                   uint16 = 183
+	PFCPASRsp_Flags                                                        uint16 = 184
+	CPPFCPEntityIPAddress                                                  uint16 = 185
+	PFCPSEReq_Flags                                                        uint16 = 186
```

Comment:
Please remove the underscore (same for some other types).

Reply:
I have removed _ from IE naming